specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/queue.json

```diff
@@ -0,0 +1,591 @@
+      "get": {
+        "tags": [
+          "QueueService"
+        ],
+        "operationId": "GetQueue",
+        "x-ms-examples": {
+          "QueueOperationGet": {
+            "$ref": "./examples/QueueOperationGet.json"
+          }
+        },
+        "description": "Gets the queue with the specified queue name, under the specified account if it exists.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QueueName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- returned table with the specified table name successfully.",
+            "schema": {
+              "$ref": "#/definitions/StorageQueue"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "QueueService"
+        ],
+        "operationId": "DeleteQueue",
+        "x-ms-examples": {
+          "QueueOperationDelete": {
+            "$ref": "./examples/QueueOperationDelete.json"
+          }
+        },
+        "description": "Deletes the queue with the specified queue name, under the specified account if it exists.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QueueName"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content - Successfully deleted the queue with the specified queue name"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/queueServices/default/queues": {
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

You also need to specify the x-ms-pageable extension ([https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable))

Reply:
done